android/src/main/java/com/twiliorn/library/CustomTwilioVideoViewManager.java

```diff
@@ -75,6 +75,9 @@ public void receiveCommand(CustomTwilioVideoView view, int commandId, @Nullable
                 boolean enableAudio = args.getBoolean(2);
                 boolean enableVideo = args.getBoolean(3);
                 boolean enableRemoteAudio = args.getBoolean(4);
+
+                //enable bluetooth by default
+                view.toggleBluetoothHeadset(true);
```

Comment:
In my opinion I would remove this line and configure bluetooth on setAudioFocus method. Then, call `stopBluetoothSco()` when the component or activity is destroyed (onHostDestroy lifecycle method).

Reply:
This method was already existing to handle bluetooth config so I didn't call the stop. But makes sense.

Reply:
Yes, but makes more sense to remove all bluetooth or methods we don't rely on more.

Reply:
BTW. `toggleBluetoothHeadset` can be call from js:  `TwilioVideoRef.setBluetoothHeadsetConnected(true)`. There is no reason to change the plugin.

Reply:
Yes, but I think @DeTeam commented that he wanted to change the plugin instead calling these methods on the JS.

Reply:
@Moreno97 I haven't removed logic  of TOGGLE BLUETOOTH HEADSET configuration if RN wants to call this in the future you can. Except the command doesn't get posted for some reason so needs investigation.